src/VisualStudio/IntegrationTest/TestSetup/TestTraceListener.cs

```diff
@@ -7,7 +7,7 @@
 
 namespace Microsoft.CodeAnalysis.ErrorReporting
 {
-    internal class WatsonTraceListener : TraceListener
+    internal class TestTraceListener : TraceListener
```

Comment:
Renamed to match the other test-specific classes, and "Watson" is a pretty out of date term here since it's not really about that anymore.